test/integration/standard_filter_test.rb

```diff
@@ -853,16 +873,20 @@ def test_all_filters_never_raise_non_liquid_exception
       ["foo", 123, nil, true, false, Drop, ["foo"], { foo: "bar" }],
     ]
     test_types.each do |first|
-      test_types.each do |other|
-        (@filters.methods - Object.methods).each do |method|
-          arg_count = @filters.method(method).arity
-          arg_count *= -1 if arg_count < 0
-          inputs = [first]
-          inputs << ([other] * (arg_count - 1)) if arg_count > 1
-          begin
-            @filters.send(method, *inputs)
-          rescue Liquid::ArgumentError, Liquid::ZeroDivisionError
-            nil
+      test_types.each do |second|
+        test_types.each do |third|
+          (@filters.methods - Object.methods).each do |method|
+            arg_count = @filters.method(method).arity
+            arg_count *= -1 if arg_count < 0
+            inputs = [first]
+            inputs << ([second] * (arg_count - 1)) if arg_count > 1
+            inputs << ([third] * (arg_count - 1)) if arg_count > 2
+
+            begin
+              @filters.send(method, *inputs)
```

Comment:
This is multiplying the number of times the filter is called with the same number of arguments when it takes less than 3 arguments, making the test significantly slower.  Now the test is taking more than a second.  I opened https://github.com/Shopify/liquid/pull/1476 to help speed up this test, generalize it and also help it test default arguments better.

Reply:
I've merged that PR to change this test to be more comprehensive and to fix the bugs that test improvement uncovered, as such, I've removed changes to this test from this PR.